jdat_notebooks/MIRI_LRS_spectral_extraction/requirements

```diff
@@ -0,0 +1,4 @@
+matplotlib==3.2.1
+numpy=1.18.4
+astropy=4.1
+jwst=0.16.1
```

Comment:
```suggestion
numpy==1.18.4
astropy==4.0.1
git+https://github.com/spacetelescope/jwst@0.16.1
```

The requirements file runs successfully with these changes. Note that I changed `astropy` from 4.1 to 4.0.1, I assume that was the intent since there is no 4.1 release.